formats/developers/6d-7d-and-8d-storage.txt

```diff
@@ -206,29 +205,41 @@ mosaic as above or to overlap e.g.
 
     E   <Plane TheC="0" TheT="12" TheZ="0" PositionX="80" PositionY="110" PositionZ="0.1"/>
 
-Sample Files
-^^^^^^^^^^^^
+Sample files
+------------
 
+Sample files are available from the
+:image_downloads:`Modulo <OME-TIFF/2015-01/modulo/>` folder of the image downloads resource.
```

Comment:
The only reason I didn't do it this time was because the schema version is *after* the file type and I wasn't sure if I could format the template correctly to get that to work.